node/testing/src/simulator/mod.rs

```diff
@@ -41,9 +42,9 @@ impl Simulator {
         {
             chain_id
         } else if let Some(node) = runner.ocaml_node(ClusterOcamlNodeId::new_unchecked(0)) {
-            node.chain_id_async().await.unwrap().into_bytes()
+            node.chain_id_async().await.unwrap()
         } else {
-            "<unknown_chain_id>".into()
```

Comment:
Since chain id calculation will be done inside the node, do we still need this or chain id configuration in rust node at all?

Reply:
Now that's gone.